app/views/compute_resources_vms/form/vmware/_network.html.erb

```diff
@@ -5,5 +5,5 @@
 <% cluster_id = params.fetch(:host, {}).fetch(:compute_attributes, {}).fetch(:cluster, nil).presence %>
 <%= select_f f, :network, vsphere_networks(compute_resource), :first, :last, { },
              :class => "col-md-3 vmware_network",
-             :size => "col-md-8", :disabled => !new_vm,
```

Comment:
Why do we need to remove this?

Reply:
I don't understand, we are supposed to remove the  `:disabled` right?

Reply:
No, that's why I'm asking why you chose to do that.

Reply:
The suggestion to remove the `:disabled => !new_vm,` option was only for testing. Now that you added the fix for this issue, we should keep it as is.

Reply:
What do you get when keeping the `disabled => !new_vm`?

Reply:
When I checked(multiple times), it was not working. It could be because of cache , I just recently reset my db so it seems to work now or could be because of any new change. But anyway, I have updated the commit. Thank you so much for all your help